extra_docs/quickstart.md

```diff
@@ -197,45 +219,48 @@ There is a special object that you need to use for aggregation:
 Example:
 ``` Javascript
 [..]
-posts.select().fields({ fn: "MAX", col: "likes" });
+posts.select()
+  .fields({ fn: "MAX", col: "likes" });
 [..]
 ```
 
 For `COUNT` function you can provide asterisk (`*`) as a field name. Also you can combine
 field names with aggregation functions to get more complicated results:
 ``` Javascript
 [..]
-posts.select().fields("date", "author", { fn: "COUNT", col: "*" });
+posts.select()
+  .fields("date", "author", { fn: "COUNT", col: "*" });
 [..]
 ``` 
 
-
-### Filtering records
+### [Filtering records](#filtering-records)
 
 You can use the filtering feature to select what records a certain query will operate on.
 
 In order to define a filter, you need to use the appropriate filtering objects. They can be created using the exposed methods `field` and `combineCriteria`. `combineCriteria` is something very specific that provides another way to create nested logical conditions. You're probably not going to use that much, but it's useful to know that it exists.
 
 ``` Javascript
-import { field } from "Anemo SDK location";
+import { field } from "jexia-sdk-js/node";
 
-let simpleCriterion = field("username").isEqualTo("Tom");
-let combinedCriteria = simpleCriterion.or(field("username").isEqualTo("Dick"));
+const simpleCriterion = field("username").isEqualTo("Tom");
+const combinedCriteria = simpleCriterion.or(field("username").isEqualTo("Dick"));
 ```
 
 In order to use these conditions, they need to be added to a query using the `.where` method
 
 ``` Javascript
 [..]
-posts.select().where(field("username").isEqualTo("Harry"));
+posts.select()
+  .where(field("username").isEqualTo("Harry"));
```

Comment:
Why not use `combinedCriteria` here? It just got introduced, so it shows to user where/how they can be used?

Reply:
to be fair I don't understand the purpose of `combineCriteria` and it isn't used anywhere. what do you say @KutsenkoA ?

Reply:
@marlosin I think that @veger is talking about variable `combinedCriteria` that was created a bit early. You might've used that variable in `select()` method, but you haven't :)

Reply:
I get that, the `import` was in the old docs, what I don't get is its purpose. why it would be good to use it? what I could do differently by calling it? I'm questioning if we should keep it in docs (or even the code) as it says _`combineCriteria` is something very specific that provides another way to create nested logical conditions_

Reply:
@marlosin I also think that `combineCriteria` class (if we are talking about it) is something internal and do not need to be exposed to a user

Reply:
ok, mention in guides removed